website/docs/getting_started/conceptual_overview.md

```diff
@@ -0,0 +1,112 @@
+If you try to build your CI/CD without this foundational understanding, you may find yourself getting confused or getting unexpected results.
+
+## Consumers and publishing pacts
+
+Whenever a consumer runs a contract test for a particular provider and it succeeds, a file is generated that records 
+all of the interactions between the consumer and the provider. This record is called a pact.
+
+In your CI/CD environment, every time you run the contract tests, the resulting pact will be published to the Pact Broker. 
+This is called a pact publication.
+
+Whenever the consumer changes, this is called a __consumer version__. The version is often identified by its git hash. 
+You may also want to add other distinguishing information, such as the feature toggle name if you are using feature 
+toggles.  In general, you want to uniquely identify each version of the consumer that may be deployed to an environment which may require a different contract.
+
+A given pact published by the consumer may or may not change actual contents of the pact (e.g. by adding a request 
+parameter, or changing the expected results).  If it does change, then the pact gets a new __pact version__.
+
+![Consumer domain model](../media/conceptual_overview_images/consumer-domain-model.png)
+
+It's important to understand a few rules about how consumers relate to pacts:
+
+- Multiple versions of the consumer can have the same pact version (if the changes to the consumer don't change the pact)
+- A specific version of the consumer will always have exactly one pact version for each provider.
+
+## Branches & Environments
+
+:::info THE GOLDEN RULE FOR LABELLING
+Label with the branch name when you publish pacts or verification results, and record deployment with the environment name after you deploy.
+
+- [branches](/pact_broker/branches)
+- [recording deployments and releases](/pact_broker/recording_deployments_and_releases)
+
+::: 
+
+At any point you may want to label a particular version of the consumer with one or more identifiers. For example, 
+you can identify the consumer with its current branch (e.g. a PR branch or the main branch), or you can identify it with the 
+environment it has been deployed to.
+
+Branches in the Pact Broker are designed to model repository (git, svn etc) branches. A branch in the Pact Broker belongs to a pacticipant (application). A branch may have many pacticipant versions, and a pacticipant version may belong to many branches (but typically, it will belong to just one).
+
+Remember that a pacticipant version in the Pact Broker should map 1:1 to a commit in your repository. To facilitate this, the version number used to publish pacts and verification results should either be or contain the commit.
+
+We recommend that you set the branch property when you publish pacts and verification results, and use record-deployment or record-release when you deploy/release.
```

Comment:
this line and the one below it feel like at odds, maybe we don't need the one above.

I feel the We recommend in stronger but the general recommendation suggests more of a community consensus rather than dictatorial

Reply:
I agree probably want to get rid of "should"